Examples/test-suite/csharp/csharp_argument_defaults_feature_runme.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using csharp_argument_defaults_featureNamespace;
+
+public class runme {
+    static void Main() {
+        Foo foo = new Foo(1);
+        foo.bar(1); //shutup compiler warning
+        Foo bar = new Foo(1, c:3);
+
+        if(bar.bar(1) != 7)
+            throw new ApplicationException("bar.bar(1) != 7");
+        if(bar.bar(1, 4, 4) != 9)
+            throw new ApplicationException("bar.bar(1, 4, 4) != 9");
+        if(bar.bar(1, y:3) != 8)
```

Comment:
Observation: this syntax requires .NET 4, but I think all our test environments are new enough.

Reply:
I assume we're talking about the optional syntax, since the point of the feature is to take advantage of that I don't see what else to do.